tests/components/logentries/test_init.py

```diff
@@ -1,82 +1,74 @@
 """The tests for the Logentries component."""
 
-import unittest
-from unittest import mock
+import pytest
 
 import homeassistant.components.logentries as logentries
 from homeassistant.const import EVENT_STATE_CHANGED, STATE_OFF, STATE_ON
-from homeassistant.setup import setup_component
+from homeassistant.setup import async_setup_component
 
-from tests.common import get_test_home_assistant
+from tests.async_mock import MagicMock, call, patch
 
 
-class TestLogentries(unittest.TestCase):
-    """Test the Logentries component."""
+async def test_setup_config_full(hass):
+    """Test setup with all data."""
+    config = {"logentries": {"token": "secret"}}
+    hass.bus.listen = MagicMock()
+    assert await async_setup_component(hass, logentries.DOMAIN, config)
+    assert hass.bus.listen.called
+    assert EVENT_STATE_CHANGED == hass.bus.listen.call_args_list[0][0][0]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)